tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -144,3 +146,12 @@ def tap_import_certs(self):
         Wait(self.marionette).until(expected.element_displayed(element))
         element.tap()
         Wait(self.marionette).until(expected.element_displayed(*self._select_certs_screen_locator))
+    
+    def tap_active_wifi(self):
+        element = self.marionette.find_element(*self._select_active_wifi_locator)
+        element.tap()
+
+    def tap_forget_wifi(self):
+        element = Wait(self.marionette).until(expected.element_present(*self._select_forget_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
```

Comment:
Here, you should also wait for the effect of the tapping the forget button, at least to make sure that now you're back to the wifi list page.  Would also be great if you can check that _select_active_wifi_locator is no longer present (since no wifi station is actively used)